socorro/unittest/external/boto/test_crashstorage.py

```diff
@@ -59,86 +40,45 @@ class ConditionallyABadDeal(Exception):
 S3ConnectionContext.conditional_exceptions = (ConditionallyABadDeal, )
 
 
-class BaseTestCase(TestCase):
-
-    @classmethod
-    def setUpClass(cls):
-        super(BaseTestCase, cls).setUpClass()
-        cls.TEMPDIR = tempfile.mkdtemp()
-
-    @classmethod
-    def tearDownClass(cls):
-        super(BaseTestCase, cls).tearDownClass()
-        shutil.rmtree(cls.TEMPDIR)
```

Comment:
I converted the classes so they aren't subclasses of `unittest.TestCase` which means we can use the pytest `tmpdir` fixture which was nice.